lessons/Algebra2/exploring-linear-models/langs/en-us/pages/graphing-models.adoc

```diff
@@ -0,0 +1,48 @@
+= Graphing Linear Models
+++++
+<style>
+
+.autonum { font-weight: bold; }
+.autonum:after { content: ')' !important; }
+
+td:first-child {
+	margin: 21px;
+	background-color: rgba(#fff, .5);
+	background-image:
+		linear-gradient(lightblue 2px, transparent 2px),
+		linear-gradient(90deg, lightblue 2px, transparent 2px),
+		linear-gradient(rgba(150, 150, 150, .3) 1px, transparent 1px),
+		linear-gradient(90deg, rgba(150, 150, 150, .3) 1px, transparent 1px);
+	background-size: 100px 100px, 100px 100px, 20px 20px, 20px 20px;
+	background-position: 56px 23px;
+}
+
+td:last-child { align-items: center; justify-items: center; }
+</style>
+++++
+
```

Comment:
This page looks MUCH better than the last time I saw it, both in terms of layout/aesthetics and its purposefulness.